backend/src/main/java/de/tobias/intestinalinspector/controller/FoodController.java

```diff
@@ -48,8 +48,8 @@ public FoodController(FoodService foodService) {
     }
 
     @PutMapping("/update/{id}")
```

Comment:
```suggestion
    @PutMapping("{id}")
```
No need to namespace the update route, the controller method is uniquely identified by the verb PUT and a route with a variable id.